src/core/public/chrome/ui/header/collapsible_nav.test.tsx

```diff
@@ -170,7 +170,7 @@ describe('CollapsibleNav', () => {
         recentlyAccessed$={new BehaviorSubject(recentNavLinks)}
       />
     );
-    expectShownNavLinksCount(component, 3);
+    expectShownNavLinksCount(component, 0);
```

Comment:
@raintygao Let's hold on merging this PR, the existing tests are for the old nav bar, changing these snapshots and tests may not be desired id we implement the workspace feature flag. Existing tests should remain the same if we test with workspace feature turned off.

Reply:
Got it, we need to implement a workspace feature flag, so it seems that we may need two snapshots instead of modifying on a single one to achieve this situation.

Reply:
Right, ideally we don't need to touch the old tests

Reply:
It's my bad.I didn't realize this before.   @SuZhou-Joe FYI for UT fix part.